locales/fr/fr.json

```diff
@@ -62,11 +62,11 @@
     "Are you sure you want to detach this resource?": "Êtes-vous sûr(e) de vouloir détacher cette donnée ?",
     "Are you sure you want to force delete the selected resources?": "Êtes-vous sûr(e) de vouloir forcer la suppression des données sélectionnées ?",
     "Are you sure you want to force delete this resource?": "Êtes-vous sûr(e) de vouloir forcer la suppression de cette donnée ?",
-    "Are you sure you want to log out?": "Are you sure you want to log out?",
+    "Are you sure you want to log out?": "Êtes-vous sûr de vouloir vous déconnecter ?",
     "Are you sure you want to restore the selected resources?": "Êtes-vous sûr(e) de vouloir restaurer les données sélectionnées ?",
     "Are you sure you want to restore this resource?": "Êtes-vous sûr(e) de vouloir restaurer cette donnée ?",
     "Are you sure you want to run this action?": "Êtes-vous sûr(e) de vouloir lancer cette action ?",
-    "Are you sure you want to stop impersonating?": "Are you sure you want to stop impersonating?",
+    "Are you sure you want to stop impersonating?": "Voulez-vous vraiment arrêter de vous faire passer pour un personnage ?",
```

Comment:
Êtes-vous sûr de vouloir arrêter de vous faire passer pour une autre personne ? 

Êtes-vous sûr de vouloir arrêter d'utiliser un autre compte ?

Reply:
je serais plus pour autre compte

Reply:
Hi @NicoBernardet and @caouecs , sorry I am not expert on it, so I will remove it from my PR. Thanks for ur help.

Reply:
No, I can fix if you want ;)